content/using-atom/sections/github-package.md

```diff
@@ -82,10 +83,24 @@ If you no longer want to keep some changes, you can discard them. It's similar t
 - **Hunk**: Click on the trash icon in the top bar of a hunk.
 - **Lines**: Right-click on a line (or multiple) and choose "Discard Selection".
 
-If you want to cancel discarding some changes, you can restore them, using the same context menu. It's similar to cherry-picking, where "their" changes are the ones you previously discarded.
-
 ![Discard changes](../../images/github-discard.png "Discard changes")
 
+
+#### Undo discard
+
+If you want cancel discarding some changes, you can restore them.
+
+Click the <kbd>...</kbd> menu in the "Unstaged Changes" header and choose "Undo Last Discard".  
+If there are any conflicts when undoing the last discard, you will be notified that "Undoing will result in conflicts".
+
+- **Merge with conflict markers**: The discarded changes are merged back into their original file.  
+The conflict resolution works similarly to cherry-picking, where "their" changes are the ones you previously discarded.
+- **Open in new file**: The discarded changed are merged with the existing contents into a new temporary file.
+- **Cancel**: The "Undo Last Discard" action is cancelled.
+
+![Handle conflicts with undo discard](../../images/github-undo-discard-conflicts.png "Handle conflicts with undo discard")
```

Comment:
This Windows-style screenshot needs updating to match the rest of the documentation.